pyproject.toml

```
@@ -30,6 +30,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 space_packet_parser = ">=4.0.2,<5"
+pandas = ">=2.0.0,<3"
+openpyxl = ">=3.0.7,<4"
 
 # Optional dependencies
 black = {version="^23.1.0", optional=true}
```

This adds pandas to openpyxl to optional list and calls it in extras called `tools`
```suggestion
black = {version="^23.1.0", optional=true}
pre-commit = {version="^3.3.3,<4", optional=true}
pydata-sphinx-theme = {version="*", optional=true}
pytest = {version=">=6.2.5,<7", optional=true}
ruff = {version="^0.0.253", optional=true}
sphinx = {version="*", optional=true}
myst-parser = {version="^2.0.0", optional=true}
pandas ={version=">=2.0.0,<3", optional=true}
openpyxl = {version=">=3.0.7,<4", optional=true}

[tool.poetry.extras]
dev = ["black", "pre-commit", "ruff"]
doc = ["myst-parser", "pydata-sphinx-theme", "sphinx"]
test = ["pytest"]
tools= ["pandas", "openpyxl"]
```

I hope that helps